CODEOWNERS

```diff
@@ -71,7 +74,7 @@ package.json @brunobar79 @estebanmino @nickbytes
 react-native.config.js @brunobar79 @estebanmino
 README.md @brunobar79 @estebanmino @nickbytes
 shim.js @brunobar79 @osdnk @estebanmino @nickbytes
-tsconfig.json @brunobar79 @osdnk @estebanmino @nickbytes
+tsconfig.json @brunobar79 @osdnk @estebanmino @nickbytes @perunt
```

Comment:
I think we should remove the addition of another codeowner to `tsconfig.json`. While I trust @perunt to make changes, it doesn't seem like `tsconfig.json` is something that will need to be updated frequently and changes should be carefully considered for their impact to the team. Having fewer codeowners for a file like this is a "feature" IMO.